src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AmazonWebServicesCredentialsBinding.java

```diff
@@ -94,6 +108,11 @@ public String getSecretKeyVariable() {
         return secretKeyVariable;
     }
 
+    @NonNull
+    public String getSessionTokenVariable() {
+        return sessionTokenVariable;
+    }
```

Comment:
Add a DataBoundSetter annotated setter, and revert your constructor changes.

```suggestion
    @NonNull
    public String getSessionTokenVariable() {
        return sessionTokenVariable;
    }
    
    @DataBoundSetter
    public void setSessionTokenVariable() {
    	this.sessionTokenVariable = StringUtils.defaultIfBlank(sessionTokenVariable, DEFAULT_SESSION_TOKEN_VARIABLE_NAME);
    }
```

Reply:
@Vlatombe Are you ok with that?